airflow-core/src/airflow/executors/executor_loader.py

```diff
@@ -74,11 +74,11 @@ def _get_executor_names(cls) -> list[ExecutorName]:
 
         all_executor_names: list[tuple[str | None, list[str]]] = cls._get_team_executor_configs()
 
-        executor_names = []
+        executor_names: list[ExecutorName] = []
         for team_name, executor_names_config in all_executor_names:
             executor_names_per_team = []
-            for name in executor_names_config:
-                if len(split_name := name.split(":")) == 1:
+            for executor_name_str in executor_names_config:
```

Comment:
do we need the `_str` here? ignore me if it's needed 👀

```suggestion
            for executor_name in executor_names_config:
```

Reply:
Attempted this before but then `executor_name` is re-used in line 129ff in the loop and generated a clash there. First I renamed below but then realized that the name below is a very good fit - so decided rather finding an alternative here.